Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -310,16 +350,34 @@ public extension Node where Context == HTML.TextAreaContext {
         .attribute(named: "rows", value: String(rows))
     }
     
+    /// Assigns a placeholder to the text area.
```

Comment:
Nitpick, feel free to ignore, but to make the documentation consistent, I believe this should be "Assign", not "Assigns".

Reply:
Fixed in e4a92dd83e2515e7fc2bafe00f999717e18f4cd9